core/dbt/main.py

```diff
@@ -998,12 +998,23 @@ def parse_args(args, cls=DBTArgumentParser):
     # if set, extract all models and blocks with the jinja block extractor, and
     # verify that we don't fail anywhere the actual jinja parser passes. The
     # reverse (passing files that ends up failing jinja) is fine.
+    # TODO remove?
```

Comment:
👍

Reply:
I might wait to remove it till the integration test PR since it's used in a way that I should also remove some plumbing in our tests while I'm in there.